### ISSUE TITLE
Stylesheets

### ISSUE THREAD

**Issue by jalliet** [tomiwaorimoloye/Portfolio#1]
Hi, how did you make sure that in _index.html_ the stylesheets weren't conflicting? Just curious

**Comment by tomiwaorimoloye** [comment]
Hey, I only use one actual css stylesheet which is found in the styles folder. The other stylesheets in the head of index.html are just CDN linkss for icons and fonts. I hope I answered your question.

**Comment by jalliet** [comment]
Yeah in part. I believed also that having too many of those CDN links could introduce conflicting styles as well. Is that necessarily true?

**Comment by tomiwaorimoloye** [comment]
Hmm. Maybe. 
If those CDNs were like a bootstrap CDN (https://getbootstrap.com/) or some other link that has it's own set of CSS. Then definitely there would be inevitable conflict. One way to partially solve this problem is to put your own CSS link at the last line of the head so that your CSS can override the other CDNs because HTML/CSS is read from top to bottom. 
But in my case, my CDNs deal with fonts and icons which are very different from CSS, so I won't have any sort of conflict.
